_dashboards/index.md

```diff
@@ -11,15 +11,56 @@ redirect_from:
 
 {%- comment -%}The `/docs/opensearch-dashboards/` redirect is specifically to support the UI links in OpenSearch Dashboards 1.0.0.{%- endcomment -%}
 
-# OpenSearch Dashboards
+# Getting started with OpenSearch Dashboards
 
-OpenSearch Dashboards is the default visualization tool for data in OpenSearch. It also serves as a user interface for many of the OpenSearch plugins, including security, alerting, Index State Management, SQL, and more.
+OpenSearch Dashboards is the preferred visualization platform for analyzing data in OpenSearch. It serves as the user interface for many openSearch features, including security, alerting, index state management, SQL, and more.
 
+Out-of-the-box OpenSearch Dashboards includes several tools for discovering and visualizing data, and OpenSearch provides a full suite of plugins that you can use to monitor events, identify trends, and automate recurring activities.
 
-## Get started with OpenSearch Dashboards
+![Sample dashboards](../images/dashboards-example.png)
 
-1. After starting OpenSearch Dashboards, you can access it at port 5601. For example, http://localhost:5601.
-1. Log in with the default username `admin` and password `admin`.
-1. Choose **Try our sample data** and add the sample flight data.
-1. Choose **Discover** and search for a few flights.
-1. Choose **Dashboard**, **[Flights] Global Flight Dashboard**, and wait for the dashboard to load.
+# Try OpenSearch Dashboards
+You can try OpenSearch Dashboards' features in the [OpenSearch Dashboards Playground](https://playground.opensearch.org/app/home).
+
+Choose **OpenSearch Dashboards > Dashboards > Create dashboard** and explore the sample Global Flight, Web Traffic, and Revenue dashboards.
+
+![Dashboards playground screenshot](../images/dashboards-playground.png)
+
+# OpenSearch Dashboards for your use case
+
+OpenSearch Dashboards makes it easy to visualize data for a number of use cases such as log analytics, application search, enterprise search, and more. Here are a few examples.  
+
+<table>
```

Comment:
Use this table instead. It will center the text and images within cells:
<table>
<tbody>
<tr>
<td style="text-align: center; vertical-align: top;">Visualize log and trace data with interactive log analytics<img src="{{site.url}}{{site.baseurl}}/images/trace_log_correlation-web.jpg" alt="Log analytics"/></td>
<td style="text-align: center; vertical-align: top;">Centralize and analyze logs for real-time threat detection and incident management<img src="{{site.url}}{{site.baseurl}}//images/detection.png" alt="Anomaly detection"/></td>
</tr>
<tr>
<td style="text-align: center; vertical-align: top;">Quickly find relvant data within your applications, websites, and data catalogs<img src="{{site.url}}{{site.baseurl}}/images/drag-and-drop.png" alt="Data overview" /></td>
<td style="text-align: center; vertical-align: top;">Find and fix problems and improve application health at a glance<img src="{{site.url}}{{site.baseurl}}/images/dashboards5.PNG" alt="Event analysis" /></td>
</tr>
</tbody>
</table>